django/website/main/templates/base.html

```diff
@@ -23,20 +23,18 @@ <h1 class="logo"><a href="/">{% settings_value "SITE_NAME" %}</a></h1>
         {% block logframe_switching_form %}
         {% if request.user and request.user.is_authenticated %}
         <form id="logframes-switch" action="{% url "switch-logframes" %}" method="post">{% csrf_token %}
-            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();"{% if logframe_list|length == 1 %}disabled="disabled"{% endif %}>
+            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();">
             	{% for logframe in logframe_list %}<option value="{{ logframe.id }}"{% if request.user.preferences.last_viewed_logframe == logframe %} selected="selected"{% endif %}>{{ logframe.name }}</option>{% endfor %}
+            	<option value="-1">{% trans "+ Create a new logframe" %}</option>
```

Comment:
Capitalisation of Logframe again

I am not sure if the + symbol should be included in the translation or not or if it should be styled in CSS. The approach you've taken would support RTL languages but we could do that in CSS anyway. I don't know what the experience would be for someone with a screen reader.

I think this selector needs a UX review 

If I am looking at an existing logframe, select "Create a new logframe" and then hit the back button, the selector still displays "Create a new logframe" and I can't select it again.

The selector disappears when I'm creating a logframe (which is part of the reason I hit the back button)